streams/src/test/java/org/apache/kafka/streams/integration/CleanUpIntegrationTest.java

```diff
@@ -62,14 +63,16 @@
     private static final String APP_ID = "cleanup-integration-test";
     private static final String INPUT_TOPIC = "inputTopic";
     private static final String OUTPUT_TOPIC = "outputTopic";
+    private static final String OUTPUT_TOPIC_2 = "outputTopic2";
     private static final String INTERMEDIATE_USER_TOPIC = "userTopic";
 
-    private static final long consumerTimeout = 2000L;
+    private static final long CONSUMER_TIMEOUT = 2000L;
```

Comment:
Perhaps use two separate timeout variables, one for Kafka Streams consumer group and one for the CleanUpClient consumer group?  You may also skip the comments "make sure ... consumer group times out" by doing so.